src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsFandomVideoPage.java

```diff
@@ -0,0 +1,48 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.contentpatterns.AdsFandomContent;
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.annotations.InBrowser;
+import com.wikia.webdriver.common.core.drivers.Browser;
+import com.wikia.webdriver.common.core.helpers.Emulator;
+import com.wikia.webdriver.common.dataprovider.ads.FandomAdsDataProvider;
+import com.wikia.webdriver.common.templates.fandom.AdsFandomTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsFandomObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsJWPlayerObject;
+
+import org.testng.annotations.Test;
+
+public class TestAdsFandomVideoPage extends AdsFandomTestTemplate {
+  @Test(groups = "AdsVideoPageF2Desktop")
+  public void adsVideoPageAdsDesktop() {
+    String testedPage = urlBuilder.globallyEnableGeoInstantGlobalOnPage(FandomAdsDataProvider.VIDEO_PAGE_SLUG,
+        FandomAdsDataProvider.INSTANT_GLOBAL_MIDROLL);
+    testedPage = urlBuilder.globallyEnableGeoInstantGlobalOnPage(testedPage,
+        FandomAdsDataProvider.INSTANT_GLOBAL_POSTROLL);
+
+    AdsFandomObject pageObject = loadVideoPage(testedPage);
+    AdsJWPlayerObject jwPlayerObject = new AdsJWPlayerObject(driver);
+
+    jwPlayerObject.verifyPreroll();
+    verifySlots(pageObject);
+  }
+
+  @InBrowser(
+      browser = Browser.CHROME,
+      emulator = Emulator.GOOGLE_NEXUS_5
+  )
+  @Test(groups = "AdsVideoPageF2Mobile")
+  public void adsVideoPageAdsMobile() {
+    adsVideoPageAdsDesktop();
```

Comment:
This doesn't look right: a test for mobile devices uses method called `adsVideoPageAdsDesktop` ;) How about making `adsVideoPageAdsDesktop` general and renaming it to `adsVideoPageAds` and then having two proxy methods with the correct annotations for desktop and mobile?

Reply:
I based on standards in other tests but yeah, it's always great moment to start changing somethings :) Renamed.